xml/decoders/nmra/cv21.22_F15.xml

```diff
@@ -25,6 +25,12 @@
       </personname>
     </author>
   </authorgroup>
+  <revision>
+      <revnumber>4</revnumber>
+      <date>2018-03-11</date>
+      <authorinitials>WK</authorinitials>
+      <revremark>Dutch translating inserted</revremark>
+    </revision>
```

Comment:
@Re465 This revision 4 block should be placed after the next line, so inside the ''&lt;revhistory&gt;'' element

Reply:
Thanks, silverailscolo, I am just home of work